apis/doc.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func init() {
-	for name, service := range services {
+	for name, service := range Services {
```

Comment:
I think you can revert this change and the change to `services.go` now that you're not using that data in `task_log.go`, right?